webpack.config.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+const webpack = require("webpack");
+const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
+const { merge } = require("webpack-merge");
+
+const bnJsReplaces = [
```

Comment:
I created this base webpack config so that all the crazyness with polyfills can be in one place.  Most of this will be able to be deleted with #2273 when `stream-browserify` and `crypto-browserify` is eliminated.